code_challenges/code_wars/hacked.py

```diff
@@ -0,0 +1,33 @@
+def find_hack(student_grades):
+    checked_scores = {}
+    hacked_students = []
+    for student in student_grades:
+        checked_scores[student[0]] = [student[1]]
```

Comment:
How can you avoid student[0] and student[1] here?

Reply:
pull it out into variables?